spec/models/admin_user_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+describe AdminUser, :type => :model do
+    
+  before(:each) do 
+    @admin  = build(:admin_user)
```

Comment:
Don't we have `subject` object automatically created for this spec?  It feels cleaner to use that rather than `@admin`